src/testing/TGV_scaling.cpp

```diff
@@ -44,8 +44,8 @@ int EulerTaylorGreenScaling<dim, nstate>::run_test() const
 
     std::ofstream myfile (all_parameters_new.energy_file + ".gpl"  , std::ios::trunc);
     const unsigned int poly_degree_start= all_parameters->flow_solver_param.poly_degree;
-
```

Comment:
avoid committing stuff like this that doesnt change the code, it can lead to more merge conflicts down the line